net-proxy/proxychains_ng/proxychains_ng-4.14.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="A tool to use SOCKS or HTTP proxy with programs that don't support it"
+DESCRIPTION="
+  ProxyChains is a UNIX program, that hooks network-related libc functions\
+  in DYNAMICALLY LINKED programs via a preloaded DLL (dlsym(), LD_PRELOAD)\
+  and redirects the connections through SOCKS4a/5 or HTTP proxies.\
+  It supports TCP only (no UDP/ICMP etc).
+
+  The way it works is basically a HACK; so it is possible that it doesn't\
+  work with your program, especially when it's a script, or starts\
+  numerous processes like background daemons or uses dlopen() to load\
+  "modules" (bug in glibc dynlinker).\
+  It should work with simple compiled (C/C++) dynamically linked programs\
+  though.
+	"
+HOMEPAGE="https://github.com/rofl0r/proxychains-ng"
+SOURCE_URI="https://github.com/rofl0r/proxychains-ng/archive/v4.14.tar.gz"
+CHECKSUM_SHA256="ab31626af7177cc2669433bb244b99a8f98c08031498233bb3df3bcc9711a9cc"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2013-2019 rolf0r"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+PATCHES="proxychains_ng-$portVersion.patch"
```

Comment:
Creating a patchset should be good, this way the credits for the patchset will be given to you